### DISCUSSION TITLE
Terms for commercial use

### DISCUSSION THREAD

**Discussion by tapmatix** [flarive/Neumorphism.Avalonia#10] [Q&A]
Hi flarive,

we are considering to use your awesome design for a commercial product.
Can you please let me know, how to get in contact?

Thanks

Thomas

**Comment by flarive** [comment]
Hello,

Yes, with pleasure... Could you tell me more about your project at flarive[@]hotmail.com ?

**Comment by cesarchefinho** [comment]
also I am considering use your theme in my comercial project.(an mobile app for social)

can you say your comercial license terms and costs ?

thanks

**Reply by flarive** [reply]
Hello,

Yes, with pleasure... Could you tell me more about your project at flarive[@]hotmail.com ?

**Reply by cesarchefinho** [reply]
uou can email me at cesarchefinho@gmail.com
